cmake/KisoLibsConfig.cmake

```
@@ -99,7 +98,7 @@ if(NOT KISO_STATIC_CONFIG)
                 configure_file(${ABS_APP_CONFIG_PATH}/${HEADER} ${DEST} COPYONLY)
             endforeach(HEADER ${USER_CONF_FILES})
         endif(NOT EXISTS ${ABS_APP_CONFIG_PATH})
-    endif(NOT APP_CONFIG_PATH)
+    endif(NOT APP_CONFIG _PATH)
```

```suggestion
    endif(NOT APP_CONFIG_PATH)
```
typo, ...not dramatic, as the contents of `endif(...)` are ignored anyway :)